### ISSUE TITLE
Renaming macro fails to activate

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#757]
_[Trac time 20090820 162915Z]_ Renaming a macro and immediately attempting to activate it fails with:

```
Traceback (most recent call last):
  File "/home/rob/src/peppy/peppy/plugins/macro.py", line 1027, in OnActivate
    wx.CallAfter(macro.playback, self.getFrame(), self.mode)
AttributeError: 'NoneType' object has no attribute 'playback'
```

However, dismissing the springtab and popping it up again works correctly.

**Comment by robmcmullen** [comment]
_[Trac time 20090820 163123Z]_ Because it still thinks it's the old name:

```
macro.py:1055 MacroListMinorMode.OnEndEdit: old=Python/aoeuaoeu new=Python/qzwm
macro.py:1024 MacroListMinorMode.OnActivate: Activating macro Python/aoeuaoeu
```

**Comment by robmcmullen** [comment]
_[Trac time 20090820 175010Z]_ 51e12f4286c1e1947cfb819c7e3b6c91383cd6bb Fixed #757: in OnEndEdit, added call to SetPyData to update full path of renamed macro
